apps/aecore/src/aec_conductor.erl

```diff
@@ -385,14 +385,14 @@ handle_worker_reply(Pid, Reply, State) ->
     Blocked = State#state.blocked_tags,
     case orddict:find(Pid, Workers) of
         {ok, Info} ->
-            cleanup_after_worker(Info),
+            cleanup_after_worker(Pid, Info),
             Tag = Info#worker_info.tag,
             State1 = State#state{workers = orddict:erase(Pid, Workers),
                                  blocked_tags = ordsets:del_element(Tag, Blocked)
                                 },
             worker_reply(Tag, Reply, State1);
         error ->
-            epoch_mining:error("Got unsolicited worker reply: ~p",
+            epoch_mining:info("Got unsolicited worker reply: ~p",
```

Comment:
finally 👯‍♂️ 🍌 

tracked in https://www.pivotaltracker.com/story/show/155776152

Reply:
Yeah, sorry...